lib/auth/touchid/api_darwin.go

```diff
@@ -154,6 +173,11 @@ func (touchIDImpl) ListCredentials() ([]CredentialInfo, error) {
 	defer C.free(unsafe.Pointer(errMsgC))
 
 	infos, res := readCredentialInfos(func(infosOut **C.CredentialInfo) C.int {
+		// ListCredentials lists all Keychain entries we have access to, without
+		// prefix-filtering labels, for example.
+		// Unexpected entries are removed via readCredentialInfos. This behavior is
+		// intentional, as it lets us glimpse into otherwise inaccessible Keychain
+		// contents.
```

Comment:
why do we want this glimpse, for debug logs?

Reply:
Yes, debug logs so we can figure out any unexpected situations that might happen. Apple docs aren't super comprehensive and I found out at least one unexpected key already, so it's a case of better safe than sorry.